### DISCUSSION TITLE
doc/intro/basic/activiti/%E6%B5%81%E7%A8%8B%E9%83%A8%E7%BD%B2

### DISCUSSION THREAD

**Discussion by giscus** [zuihou/zuihou-admin-cloud#315] [企业版]
# doc/intro/basic/activiti/%E6%B5%81%E7%A8%8B%E9%83%A8%E7%BD%B2

微服务中后台快速开发平台

https://tangyh.top/doc/intro/basic/activiti/%E6%B5%81%E7%A8%8B%E9%83%A8%E7%BD%B2.html

**Comment by felix-liang** [comment]
想问一下这里有什么原因只有COLUMN模式和NONE模式才有Activiti工作流，DATASOURCE模式没有。
有计划把工作流加到DATASOURCE模式吗

**Reply by zuihou** [reply]
Activiti 不支持 多数据源。
